lib/publiccloud/provider.pm

```diff
@@ -333,11 +331,11 @@ sub terraform_apply {
         assert_script_run('cd ' . TERRAFORM_DIR . "/$cloud_name");
         my $sap_media            = get_required_var('HANA');
         my $sap_regcode          = get_required_var('SCC_REGCODE_SLES4SAP');
-        my $ha_sap_repo          = get_required_var('HA_SAP_REPO');
         my $storage_account_name = get_var('STORAGE_ACCOUNT_NAME');
         my $storage_account_key  = get_var('STORAGE_ACCOUNT_KEY');
         my $sle_version          = get_var('FORCED_DEPLOY_REPO_VERSION') ? get_var('FORCED_DEPLOY_REPO_VERSION') : get_var('VERSION');
         $sle_version =~ s/-/_/g;
+        my $ha_sap_repo = get_var('HA_SAP_REPO') ? get_var('HA_SAP_REPO') . '/SLE_' . $sle_version : '';
```

Comment:
but you removed `%HA_SAP_REPO%` everywhere in tfvars files . so with current state of this files there is no point to pass it at all. or am I missing something ?

Reply:
Good point, that a mistake, `%HA_SAP_REPO%` should be in the tfvars file. But with an empty value by default, this is why I didn't had any error during my tests.